odoo/tests/common.py

```diff
@@ -465,6 +466,15 @@ def tearDown(self):
         super(SavepointCase, self).tearDown()
 
 
+def terminate_chrome(chrome_pid):
+    try:
+        os.kill(chrome_pid, 0)
```

Comment:
You can call os.kill(chrome_pid, signal.SIGTERM)
This will avoid a possible corner case where the process disapear between the two kill.